apps/frontend/src/App.tsx

```diff
@@ -45,7 +54,11 @@ function App() {
             </Container>
           </Flex>
           <Footer />
-
+          <Alert status='warning'>
+            <AlertIcon />
+            <AlertTitle>Disclaimer</AlertTitle>
+            <AlertDescription>The use case of this template is inspired by the <Link color='teal.500' className="hover:underline text-blue-500" href={"https://www.greencart.vet/"}>GreenCart Dapp</Link>. The code has been built from scratch and does not contain any references to the GreenCart codebase.</AlertDescription>
```

Comment:
tailwind is not used

Reply:
Didn't see 😅. Corrected: https://github.com/vechain/x-app-template/pull/12/commits/d24fd89209a22542ee078de948350a6b60857376